alg/teca_deeplabv3p_ar_detect.py

```diff
@@ -401,7 +401,7 @@ def build_model(self, state_dict_deeplab_file=None):
 
         state_dict_deeplab = self.load_state_dict(state_dict_deeplab_file)
 
-        model = DeepLabv3_plus(n_classes=1, _print=False)
+        model = teca_deeplabv3p_ar_detect_internals.DeepLabv3_plus(n_classes=1, _print=False)
```

Comment:
does this still run cleanly through pycodestyle? I guess this line it too long, perhaps there are others

Reply:
No it doesn't. Cleaning it up now